app/views/about/index.html.erb

```diff
@@ -116,7 +116,7 @@
     </div>
   </section>
 
-  <% unless is_user_a_guest? %>
+  <%# unless is_user_a_guest? %>
```

Comment:
nit: can we remove this instead of commenting it out?

Reply:
Got it.